handbook/product/user_feedback.md

```diff
@@ -47,7 +48,7 @@ We have a few different email lists that are used to send us feedback.
 ### Slack
 
 - **Purpose:** Slack is a fast way for existing customers and Sourcegraph teammates to provide feedback. 
-- **Owner:** Most feedback in Slack appears in or gets forwarded into a relevant channel that already has clear owners. For feedback that does not have an obvious other location, whoever comes across it should post to the `#feedback` Slack channel.This channel is owned by the Product team. 
+- **Owner:** Most feedback in Slack appears in or gets forwarded into a relevant channel that already has clear owners. For feedback that does not have an obvious other location, whoever comes across it should post to the `#feedback` Slack channel. This channel is owned by the Product team. 
```

Comment:
Question about #feedback versus team channels exists here, as well—is it always #feedback, or just sometimes if no clear owner exists? Standardizing this would help, I think.

Reply:
https://sourcegraph.slack.com/archives/C0C324C91/p1618558959248400?thread_ts=1618467602.243500&cid=C0C324C91